base/deprecated.jl

```diff
@@ -259,3 +259,5 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool()      rand(Bool)
+@deprecate randbool(x...)  bitrand(x)
```

Comment:
Oh, you're right, did not realize `randbool()` with no inputs gave a `Bool`, nevermind. Glad we're deprecating that.